www/src/components/pdf/ViewerSDKClient.ts

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/*
+Copyright 2020 Adobe
+All Rights Reserved.
+
+NOTICE: Adobe permits you to use, modify, and distribute this file in
+accordance with the terms of the Adobe license agreement accompanying
+it. If you have received this file from a source other than Adobe,
+then your use, modification, or distribution of it requires the prior
+written permission of Adobe.
+*/
+import axios from 'axios';
+
+import arrayBufferToBase64 from '../../util/helpers';
+declare global {
+    interface Window {
+        AdobeDC: any;
+    }
+}
+
+type saveAPIResponse = {
+    code: number;
+    data: {
+        metaData: any;
+    };
+};
+
+class ViewSDKClient {
+    readyPromise: Promise<void>;
+    adobeDCView: any;
+
+    constructor() {
+        this.readyPromise = new Promise<void>((resolve) => {
+            if (window.AdobeDC) {
+                console.log(this.readyPromise);
+
+                resolve();
+            } else {
+                console.log('yooo');
+                /* Wait for Adobe Document Services PDF Embed API to be ready */
+                document.addEventListener('adobe_dc_view_sdk.ready', () => {
+                    console.log('about to resolve');
```

Comment:
Are these logs just TODOs? What is the plan here?

Reply:
lmao mb those are remnants from me debugging... will remove